scripts/buildtools/commands/verify-package.js

```diff
@@ -17,7 +17,7 @@ const EXPECTED_BUNDLE_SIZES = {
   'okta-sign-in.oie.min.js': 1.2 * MB,
   'okta-sign-in.polyfill.js': 504 * KB,
   'okta-sign-in.polyfill.min.js': 108 * KB,
-  'okta-sign-in.next.js': 7 * MB, // NOTE: bloated temporarily without tree-shaking
+  'okta-sign-in.next.js': 1.6 * MB, // NOTE: bloated temporarily without tree-shaking
```

Comment:
Oh, much smaller now! What happened?

Reply:
Oh yea, this _has_ to be because of the addition of vite, I recall when Lester added vite and used it to build it decreased the js file size down drastically. Since adding vite, we never came back and updated this verify package script so this is the first time its being reflected 🙏